### DISCUSSION TITLE
Incubation Phase

### DISCUSSION THREAD

**Discussion by vivektk84** [apache/pekko#534] [General]
Hi,
I would like to start using PEKKO as alternative to AKKA, however not sure of the "incubating" status. 
Is there a specific timeline where the PEKKO project remains in incubation phase? Is there a roadmap defined already?

**Comment by justinmclean** [comment]
Incubating means learning to become an ASF project and building a community around the project. It doesn't reflect the projects functionality or code quality. Most projects graduate with 1 1/2 years of entering Incubation, some take 2 or 3 years or more, and some do not graduate. What concerns you about using an incubating project?

**Reply by vivektk84** [reply]
I wanted to migrate from Akka to use Pekko for an enterprise product. Hence I would like to know what is the impact of "incubation" on the usage in terms of license.

**Reply by He-Pin** [reply]
Hi, I think the code quality is good, and you can do any contribution if you would like to (in case some feature is missing). I agree with  @justinmclean , the code is not a problem and the license is apache will not cause any problem too.

**Reply by He-Pin** [reply]
What's your Akka version in used, if you are using Akka version 2.7.x+, then there may some features are currently missing.

**Reply by justinmclean** [reply]
Incubation has no impact on the license. Pekko's releases are available under the open source Apache 2.0 license. The Apache 2.0  license is permissive and can be used for in an enterprise product, as long as you abide by the terms of the license and respect the ASF's trademarks. Multiple people (and tools) have checked each Pekko release for possible licensing issues.

**Reply by vivektk84** [reply]
Thanks @justinmclean for the information.
@He-Pin , I use 2.6.x version of Akka currently.
